src/app/body/pipeline/pipeline.component.ts

```diff
@@ -10,23 +10,29 @@ import { StartServiceService } from "src/app/services/start/start-service.servic
 import { AngularFireFunctions } from '@angular/fire/compat/functions';
 import { TeamServiceService } from 'src/app/services/team/team-service.service';
 import { NavbarHandlerService } from 'src/app/services/navbar-handler/navbar-handler.service';
+import { GitlabComponent } from '../task-details/gitlab/gitlab.component';
+import { GitlabServiceService } from 'src/app/services/gitlab-service/gitlab-service.service';
 
 @Component({
   selector: 'app-pipeline',
   templateUrl: './pipeline.component.html',
   styleUrls: ['./pipeline.component.css']
 })
 export class PipelineComponent {
-  componentName = "PIPELINE"
+  componentName = "PIPELINE";
   workflows;
+  workflows2;
```

Comment:
What is this reference?

Reply:
yes sir but if we change the team it's not updating the page instantly if we use a single object to show data that's why i  used two different objects to show data